ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml

```diff
@@ -262,15 +262,12 @@ tests:
     cluster_profile: hypershift
     env:
       CLUSTER_TYPE_OVERRIDE: openstack-nfv
-      HYPERSHIFT_ZONES: us-east-1a,us-east-1b,us-east-1c
```

Comment:
I don't see this defined in the new workflow. Is this just unnecessary configuration that we're removing? Ditto for the other files that have the same change.

Reply:
The default in `ci-operator/step-registry/hypershift/setup-nested-management-cluster/hypershift-setup-nested-management-cluster-chain.yaml` is safe and we should not have to override it in any of our jobs. I have found we have it else where, I'll clean that up outside this PR.